packages/mobile-app/app/(tabs)/index.tsx

```diff
@@ -32,6 +37,37 @@ export default function Balances() {
 
   return (
     <View style={styles.container}>
+      <View style={{ display: "flex", flexDirection: "row" }}>
+        <LinkButton href="/menu/" title="Menu" />
+        <LinkButton
+          href="/accountSelect/"
```

Comment:
nitpick/bike-shedding: This is super low priority and def not blocking.

Since RN does file-based navigation, I think of these like URL paths. Because of that, for consistency with our other projects and cause I feel like kebab cased route file names are more common, I would suggest that we kebab-case the route files

Reply:
Yeah I agree! I had that thought as well but forgot to make the change, thanks for catching this.